packages/nextjs/hooks/scaffold-eth/useTransactor.tsx

```diff
@@ -69,7 +71,8 @@ export const useTransactor = (_signer?: Signer): TTransactionFunc => {
       notificationId = notification.loading(
         <TxnNotification message="Waiting for transaction to complete." blockExplorerLink={blockExplorerTxURL} />,
       );
-      transactionReceipt = await transactionResponse.wait();
+
+      const transactionReceipt = await transactionResponse.wait(options?.blockConfirmations);
```

Comment:
If options?.blockConfirmations is undefined, will it use the default one block confirmation on wait?

Reply:
Yes yes, here in docs -> https://docs.ethers.org/v5/api/providers/types/#providers-TransactionResponse

In implementation : 
https://github.com/ethers-io/ethers.js/blob/c785c1e5153d5a85a78f90f177577f2c8df15529/src.ts/providers/abstract-provider.ts#LL1017C61-L1017C61

So if its `undefined | null` or the arg is not passed it uses 1 as the default